src/nodejs/lambda-layers/database-util/src/query/form.js

```diff
@@ -23,6 +23,12 @@ const refs = {
 
 const fields = (list) => list.map((field) => fieldMap[field]);
 
+const createForm = (params) => `
+INSERT INTO form (short_name, version, long_name, description, daac_only)
+VALUES ('${params.short_name}', ${params.version}, '${params.long_name}', ${params.description ? `'${params.description}'` : 'NULL'}, ${params.daac_only || false})
```

Comment:
Should we just let the db handle the default values here? Maybe updating instead to something like
```
INSERT INTO form (short_name, version, long_name${params.description ? ', description' : ''}${params.daac_only ? ', daac_only' : ''})
VALUES ('${params.short_name}', ${params.version}, '${params.long_name}'${params.description ? `, '${params.description}'` : ''}${params.daac_only ? `, ${params.daac_only}` : ''})
RETURNING *;
```
to be fair, this technically means if daac_only is false it won't post it to the db; however, that shouldn't be an issue since the default value is false

Reply:
I have updated the query and pushed it to the branch.